fe1-web/store/Storage.ts

```diff
@@ -25,11 +25,11 @@ export function storeInit(): PersistStoreConfig {
   };
 
   const appReducer: Reducer = persistCombineReducers(persistConfig, {
-    keyPairReducer,
-    availableLaosReducer,
-    openedLaoReducer,
-    currentEventsReducer,
-    openRollCallIDReducer,
+    getKeyPair: keyPairReducer,
+    getOpenedLao: openedLaoReducer,
+    getAvailableLaos: availableLaosReducer,
+    getCurrentEvents: currentEventsReducer,
+    getOpenRollCallId: openRollCallIDReducer,
```

Comment:
you're gonna hate me, but no need for a `get` here, since the output is a data structure anyways 😅